File: src/test/java/xyz/cssxsh/mirai/economy/EconomyServiceTestCoin.java
package xyz.cssxsh.mirai.economy;

import org.jetbrains.annotations.NotNull;
import xyz.cssxsh.mirai.economy.service.EconomyCurrency;

public class EconomyServiceTestCoin implements EconomyCurrency {

    @NotNull
    @Override
    public String getId() {
        return "EconomyUtilsTest";
    }

    @NotNull
    @Override
    public String getName() {
        return "EconomyUtilsTest";
    }

    @NotNull
    @Override
    public String getDescription() {
        return "EconomyUtilsTest";
    }

    public static EconomyServiceTestCoin INSTATE = new EconomyServiceTestCoin();
}

File: src/test/kotlin/xyz/cssxsh/mirai/economy/EconomyServiceTestCoin.kt
package xyz.cssxsh.mirai.economy

import xyz.cssxsh.mirai.economy.service.EconomyCurrency

object EconomyServiceTestCoin : EconomyCurrency {
    override val id: String = "test"
    override val name: String = "TEST 币"
    override val description: String = "...."

    override fun format(amount: Double): String = "$amount 枚 TEST 币"
}